src/OmniSharp.Abstractions/Models/v1/FindSymbols/OmniSharpSymbolFilter.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace OmniSharp.Models.FindSymbols
+{
+    [Flags]
+    public enum OmniSharpSymbolFilter
+    {
+        None = 0,
+        Namespace = 1,
+        Type = 2,
+        Member = 4,
+        TypeAndMember = 6,
```

Comment:
you can use thee same syntax as in Roslyn e.g. `TypeAndMember = Type | Member` or `All = Namespace | Type | Member` to make it more readable

Reply:
Will do so. Thanks for the feedback.